src/api/controllers/articleController.js

```diff
@@ -101,4 +102,35 @@ export default class ArticleController {
       message: 'Your Article is up-to-date now, Thanks'
     });
   }
+
+  /**
+   * allow a user to highlight a text in an article
+   *
+   * @author Alain Burindi
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async highlight(req, res) {
+    const { articleId } = req.params;
```

Comment:
@alainburindi I recognize that it's fine to be using the article Id, but I didn't see that you created a middleware to check if that article exists. 
I advise you to kindly use the article slug because we have a middleware to check if that article exists

Reply:
yeah, let's be consistent